extensions/wikia/AchievementsII/services/AchAwardingService.class.php

```diff
@@ -409,10 +410,10 @@ private function processInTrackEditCategory() {
 	private function processInTrackPicture() {
 		// BADGE_PICTURE
 		$insertedImages = Wikia::getVar( 'imageInserts' );
-		if ( !empty( $insertedImages ) && is_array( $insertedImages ) ) {
-			foreach ( $insertedImages as $inserted_image ) {
-				if ( $inserted_image['il_to']{0} != ':' ) {
-					if ( wfFindFile( $inserted_image['il_to'] ) ) {
+		if( !empty( $insertedImages ) && is_array( $insertedImages ) ) {
+			foreach( $insertedImages as $inserted_image ) {
+				if( $inserted_image[ 'il_to' ]{0} != ':' ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "il_to" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)